src/www/system_general.php

```diff
@@ -295,6 +310,16 @@
                 </div>
               </td>
             </tr>
+            <tr>
+              <td><a id="help_for_domainsearchlist" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Domain search list"); ?></td>
+              <td>
+                <input name="domainsearchlist" type="text" value="<?=$pconfig['domainsearchlist'];?>" />
+                <div class="hidden" data-for="help_for_domainsearchlist">
+                  <?=gettext("Enter a custom ordered space separated list of search domain"); ?>
```

Comment:
```suggestion
                  <?=gettext("Search list for host-name lookup. The search list is normally determined from the local domain name; by default, it contains only the local domain name. This may be changed by listing the desired domain search path. Up to 5 names can be separated using a semicolon character. Most resolver queries will be attempted using each component of the search path in turn until a match is found."); ?>
```

Taken from RESOLVER(5) man page.
Assuming the semicolon suggestion from above gets incorporated.